### PR TITLE
Rake docs

### PR DESCRIPTION
**Pull request by g4s8**
#345 - Updated readme with rake docs

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#348](https://github.com/zerocracy/datum/issues/348) is now in scope, role is `REV`

File: README.md [hunk 1]
```diff
@@ -86,9 +86,23 @@ number it upgrades an `.xml` file to. All versions are
...
+After all changes are made, don't forget to run:
+```sh
+bundle update
+rake
+```
+
+To make `rake` working you will need to install:
+ - Ruby 2.2+
+ - [rake](https://github.com/ruby/rake) (`gem install rake`)
+ - [bundler](https://bundler.io/)  (`gem install bundler`)
```
Comment (carlosmiranda) [inline README.md]: @g4s8 not sure this is a complete list. I remember having to install `xcop` gem as well.

Comment (g4s8) [reply README.md]: @carlosmiranda all deps will be installed automatically with bundler: `bunle install`, they are specified in `Gemfile`

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#348](https://github.com/zerocracy/datum/issues/348) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

No issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thanks!

**Comment by ypshenychka** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**